pkg/commands/edit/add/flagsandargs_test.go

```diff
@@ -102,3 +102,33 @@ func TestExpandFileSource(t *testing.T) {
 		t.Fatalf("FileSources is not correctly expanded: %v", fa.FileSources)
 	}
 }
+
+func TestExpandFileSourceWithKey(t *testing.T) {
+	fakeFS := fs.MakeFakeFS()
+	fakeFS.Create("dir/fa1")
```

Comment:
to make the case for globbing more obvious, make this longer, e.g.
`dir/faaaaaaaaaabbbbbbbbbccccccccccccccccc`

Reply:
updated